userspace/libscap/scap.h

```diff
@@ -144,6 +145,20 @@ typedef struct scap_stats
 	uint64_t n_tids_suppressed; ///< Number of threads currently being suppressed.
 }scap_stats;
 
+/*!
+  \brief Statistics about an in progress capture (including counters and libbpf stats, compare to `bpftool prog show` CLI).
+*/
+typedef struct scap_stats_v2
+{
+	/* Metadata */
+	char name[STATS_NAME_MAX];
+	int valid;
+	/* Stats values */
+	uint64_t u64value;
+	int64_t s64value;
```

Comment:
🙃 yes we absolutely need uint64_t for these crazy servers just like we already do for all counters. Let's not change this, also personally I doubt we will need unsigned here in the foreseeable future, so I can remove `s64value` from the schema, @gnosek more preferences?

Reply:
Out of curiosity, what's a metric that can legitimately reach 2<<63? IMO if it's a counter, the extra bit won't change much (you still have to handle rollover) and if it's not (e.g. memory usage or limit), then the value in the 2<<60+ range are just about infinite anyway ;)

Still, I'd rather have one value instead of two so if you insist on an uint64_t, let's drop the int64_t.

Reply:
🙃 great let's stick to uint64_t only then since it's counters.